.github/workflows/py-packaging.yaml

```diff
@@ -26,6 +29,7 @@ jobs:
         path: apis/python/dist/
 
   wheels:
+    # Note: tries all supported Python versions as specified in apis/python/pyproject.toml
```

Comment:
> as specified in apis/python/pyproject.toml

\<pedantic\>Currently this is actually specified in `setup.py` as `python_requires`\</pedantic\>

Reply:
@gsakkis thanks for the update! :)

Reply:
https://github.com/single-cell-data/TileDB-SOMA/commit/e5045e88412ff0c439c0c00bd6ffd795917846cf